dsp/prerequisites.md

```diff
@@ -0,0 +1,48 @@
+---
+title: DSP Introduction
+description: Prerequisites
+---
+
+[&larr; back to Overview](/dsp)
+
+# Prerequisites
+
+This page summarizes the fundamental requirements for working with the Digital Solutions Platform (DSP).
+
+Ensure that you have completed these steps before proceeding with any activity on the DSP platform.
+
+---
+
+## 1. Access Credentials
+1. **Azure Subscription**
+    - Verify you have valid Azure credentials, Azure Subscription.
+2. **GitHub Account**
+    - Ensure you have a GitHub account, connected to the Vaillant Group organization.
+3. **Organization Approvals**
+    - Confirm any internal governance or security approvals.
+
+---
+
+## 2. Necesssary CLI Tools
+
+1. **kubectl**
+    - Required for interacting with Kubernetes clusters.
+    - Installation guides: [Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/)
+
+2. **Azure CLI**
+    - Needed for managing resources in Azure.
+    - Installation guides: [Microsoft Documentation](https://learn.microsoft.com/en-gb/cli/azure/install-azure-cli)
+
+3. **Optional Tools**
+    - **Helm**: [Helm documentation](https://helm.sh/docs/intro/install/)
```

Comment:
Let's rather mention kustomize than helm here, as our sample repo is built with kustomize